runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XpGlobesTest.java

```diff
@@ -0,0 +1,168 @@
+package net.runelite.client.plugins.xpglobes;
+
+import com.google.inject.Guice;
+import com.google.inject.testing.fieldbinder.Bind;
+import com.google.inject.testing.fieldbinder.BoundFieldModule;
+import java.util.Collections;
+import javax.inject.Inject;
+import net.runelite.api.Experience;
+import net.runelite.api.Skill;
+import net.runelite.api.events.StatChanged;
+import net.runelite.client.plugins.xptracker.XpTrackerService;
+import net.runelite.client.ui.overlay.OverlayManager;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import static org.mockito.Mockito.when;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class XpGlobesTest
+{
+	@Inject
+	private XpGlobesPlugin xpGlobesPlugin;
+
+	@Mock
+	@Bind
+	private OverlayManager overlayManager;
+
+	@Mock
+	@Bind
+	private XpGlobesOverlay xpGlobesOverlay;
+
+	@Mock
+	@Bind
+	private XpTrackerService xpTrackerService;
+
+	@Mock
+	@Bind
+	private XpGlobesConfig xpGlobesConfig;
+
+	private int virtualLevelTotalXp;
```

Comment:
This value doesn't seem to be changed in any test, and should probably be a static final variable rather than initialized in `before()`:

```java
private static final int virtualLevelTotalXp = Experience.getXpForLevel(Experience.MAX_REAL_LEVEL + 1);
```

Reply:
I like that a lot. Done.